python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -35,6 +35,16 @@
 )
 
 
+def _check_groupby_supported(func):
+    def wrapper(*args, **kwargs):
+        gb = args[0]
+        if _groupby_supported(gb):
+            return func(*args, **kwargs)
+        return getattr(super(type(gb), gb), func.__name__)(*args[1:], **kwargs)
```

Comment:
Noting that this changes the behavior of the upstream Dask call when no args / kwargs are specified - now, we end up going with the defaults specified by upstream Dask rather than the defaults for dask-cudf.

I imagine this shouldn't be an issue if we intend to keep the defaults for `split_out`/`split_every` consistent between the two libraries, but it could become an issue if that changes in the future.

Reply:
> now, we end up going with the defaults specified by upstream Dask rather than the defaults for dask-cudf.

Can we add a comment about this here? Are the defaults different in dask/dask_cudf. In the long run, it would be nice for them to be the same anyway.

Reply:
> Can we add a comment about this here?

Sure!

> Are the defaults different in dask/dask_cudf.

Yeah they're currently the same (`split_every=None, split_out=1`)